.github/workflows/ci.yml

```diff
@@ -0,0 +1,229 @@
+    services:
+      postgres1:
+        image: postgres:15
+        env:
+          POSTGRES_USER: edc
+          POSTGRES_PASSWORD: edc
+          POSTGRES_DB: edc
+        ports:
+          - 54321:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      postgres2:
+        image: postgres:15
+        env:
+          POSTGRES_USER: edc
+          POSTGRES_PASSWORD: edc
+          POSTGRES_DB: edc
+        ports:
+          - 54322:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    env:
+      SKIP_TESTCONTAINERS: true
+      TEST_POSTGRES_1_JDBC_URL: jdbc:postgresql://localhost:54321/edc
+      TEST_POSTGRES_1_JDBC_USER: edc
+      TEST_POSTGRES_1_JDBC_PASSWORD: edc
+      TEST_POSTGRES_2_JDBC_URL: jdbc:postgresql://localhost:54322/edc
+      TEST_POSTGRES_2_JDBC_USER: edc
+      TEST_POSTGRES_2_JDBC_PASSWORD: edc
+    steps:
+      - uses: actions/checkout@v3
+      - name: "Set up JDK 17"
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: "Gradle: Validate Gradle Wrapper"
+        uses: gradle/wrapper-validation-action@ccb4328a959376b642e027874838f60f8e596de3
+      - name: "Gradle: Include last commit info and build date for JARs"
+        run: |
+          git log -1 > extensions/last-commit-info/src/main/resources/jar-last-commit-info.txt
+          echo $(date --utc +%FT%TZ) > extensions/last-commit-info/src/main/resources/jar-build-date.txt
+      - name: "Gradle: Overwrite Artifact Version (Release Only)"
+        run: |
+          if [ "${{ startsWith(github.ref, 'refs/tags/') }}" == "true" ]; then
+            GRADLE_ARGS="-PsovityEdcExtensionsVersion=${GITHUB_REF#refs/tags/v}"
+          fi
+          echo "GRADLE_ARGS=$GRADLE_ARGS" >> $GITHUB_ENV
+      - name: "Gradle: Build"
+        uses: gradle/gradle-build-action@749f47bda3e44aa060e82d7b3ef7e40d953bd629
+        with:
+          arguments: build ${{ env.GRADLE_ARGS }}
+      - name: "Gradle: Publish"
+        uses: gradle/gradle-build-action@749f47bda3e44aa060e82d7b3ef7e40d953bd629
+        if: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/releases/') }}
+        with:
+          arguments: publish ${{ env.GRADLE_ARGS }}
+        env:
+          USERNAME: ${{ github.actor }}
+          TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: "Upload artifact 'connector-jars'"
+        uses: alehechka/upload-tartifact@v2
+        with:
+          name: connector-jars
+          path: launchers/connectors/**/build/libs/app.jar
+  docker-image:
+    name: Docker Image
+    needs: build-gradle-project
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      matrix:
+        imageVariants: [
```

Comment:
It did before. I could further halve the build times by merging the steps and building the images in series, because docker will cache the layers and an entire minute came from sharing files between jobs